### PR TITLE
ADVANCED CRATES Rebase Edition!

### PR DESCRIPTION
**Pull request by fluffe9911**
🆑 fluffe9911 for porting Nickvr628 for coding and Partheo for Spriting
add: A new spherical crate cargo can order!
🆑
see  https://github.com/yogstation13/yogstation/pull/3168 for more details and the original pr

### CODE REVIEW

**Comment by Amelia0010** [comment]
why

**Comment by fluffe9911** [comment]
why not *shrug*

File: yogstation/code/modules/cargo/cargo_packs.dm [hunk 1]
```diff
@@ -42,3 +42,13 @@
 	cost = 8000
 	contains = list(/mob/living/simple_animal/chocobo)
 	crate_name = "chocobo crate"
+
+/datum/supply_pack/misc/sphere
+	name = "Advanced Crates"
+	desc = "Contains Advanced Crates that defie all known cargo standards!"
```
Comment (monster860) [inline yogstation/code/modules/cargo/cargo_packs.dm]: "defie"

**Comment by monster860** [comment]
time to order 10 of these as RD.

File: yogstation/code/modules/cargo/cargo_packs.dm [hunk 2]
```diff
@@ -42,3 +42,13 @@
 	cost = 8000
 	contains = list(/mob/living/simple_animal/chocobo)
 	crate_name = "chocobo crate"
+
+/datum/supply_pack/misc/sphere
+	name = "Advanced Crates"
+	desc = "Contains Advanced Crates that defy all known cargo standards!"
+	cost = 2600
```
Comment (ThatLing) [inline yogstation/code/modules/cargo/cargo_packs.dm]: set this to like a million

File: yogstation/code/modules/cargo/cargo_packs.dm [hunk 3]
```diff
@@ -42,3 +42,13 @@
...
+	contains = list(/obj/structure/closet/crate/sphere,
+					/obj/structure/closet/crate/sphere,
+					/obj/structure/closet/crate/sphere)
```
Comment (ThatLing) [inline yogstation/code/modules/cargo/cargo_packs.dm]: set this to like 1

**Review by ThatLing — DISMISSED** [review]

File: yogstation/code/modules/cargo/cargo_packs.dm (hunk 2 above)
Comment (fluffe9911) [reply yogstation/code/modules/cargo/cargo_packs.dm]: uhm alrighty

**Comment by ThatLing** [comment]
Fine to merge for me

**Comment by ThatLing** [comment]
On closer thought, I like the idea of this being a random locker somewhere on the room. I'll pass

**Pull request closed without merging**